pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA      = "n/a"
 	name        = "app-operator"
 	source      = "https://github.com/giantswarm/app-operator"
-	version     = "1.0.1"
+	version     = "1.0.2-dev"
```

Comment:
Bumping version.

Reply:
That should at best happen separately directly after a tag got pushed.

Reply:
Definitely this was an omission due to the problems with the rollout.

I need to fix the integration tests because they're trying to use appr :( Fixes are in the helm 3 branch.